### PR TITLE
lftp: updated to 4.8.2

### PR DESCRIPTION
**Pull request by fededim**
Signed-off-by: fededim@gmail.com

Compile tested: mipsel-24kc

### CODE REVIEW

File: net/lftp/Makefile [hunk 1]
```diff
@@ -8,13 +8,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=lftp
-PKG_VERSION:=4.7.5
+PKG_VERSION:=4.8.2
 PKG_RELEASE:=1
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=http://lftp.tech/ftp/ \
```
Comment (diizzyy) [inline net/lftp/Makefile]: The ftp mirror is dead, please add another mirror.
Something like: https://mirror.csclub.uwaterloo.ca/gentoo-distfiles/distfiles/ as primary (first) and http://lftp.tech/ftp/ as secondary. Also, please add a comment https://github.com/lede-project/source/pull/1133 if you want an alias and better overall availability (it'd add more mirrors).

File: net/lftp/Makefile [hunk 2]
```diff
@@ -8,13 +8,14 @@
...
 		ftp://ftp.st.ryukoku.ac.jp/pub/network/ftp/lftp/
-PKG_HASH:=53b5c5eba2e38b418bcf451776f2df596050dff4db90ab4ea73d872f8b1fd0d8
 
+PKG_MD5SUM:=36f3ca2c070391282e0260dd6234af3b
```
Comment (diizzyy) [inline net/lftp/Makefile]: Drop MD5SUM, it's deprecated

File: net/lftp/Makefile [hunk 3]
```diff
@@ -70,6 +71,8 @@ CONFIGURE_ARGS += \
 #MAKE_VARS += \
```
Comment (diizzyy) [inline net/lftp/Makefile]: Remove unnecessary lines

**Comment by diizzyy** [comment]
Also, you need a proper signoff line and remove the stray commit from your pull request

File: net/lftp/Makefile (hunk 1 above)
Comment (fededim) [reply net/lftp/Makefile]: The layout of the description seems right, what's wrong with it ?
image

File: net/lftp/Makefile (hunk 2 above)
Comment (fededim) [reply net/lftp/Makefile]: It is for compatibility with older Openwrt, does it support also PKG_HASH now ?

File: net/lftp/Makefile (hunk 3 above)
Comment (fededim) [reply net/lftp/Makefile]: Ok

**Pull request closed without merging**